src/coreclr/jit/importercalls.cpp

```diff
@@ -3433,8 +3433,7 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
             }
 #endif // defined(TARGET_ARM64) || defined(TARGET_RISCV64)
 
-#if defined(TARGET_XARCH) || defined(TARGET_ARM64) || defined(TARGET_RISCV64)
-            // TODO-ARM-CQ: reenable treating InterlockedCmpXchg32 operation as intrinsic
+#if defined(TARGET_XARCH) || defined(TARGET_ARM64) || defined(TARGET_ARM) || defined(TARGET_RISCV64)
```

Comment:
nit:
```suggestion
#if defined(TARGET_XARCH) || defined(TARGET_ARMARCH) || defined(TARGET_RISCV64)
```
(and in other places)